### ISSUE TITLE
An option to build nvc without "default" library paths

### ISSUE THREAD

**Issue by filmil** [nickg/nvc#652]
When NVC is compiled, it obtains a default setting for standard library paths. The default path is usually `$PREFIX/lib/nvc` and such.

This makes using NVC in hermetic environments like `bazel` harder than necessary. It would be nice if the library search path could be made empty by default, requiring the `-L` switch. 
This works much better with `bazel`, which assumes no implicit environment, to achieve hermetic reproducible builds.

**Comment by nickg** [comment]
I added a `--disable-default-paths` configure option. Can you see if that does what you want?

**Comment by filmil** [comment]
I will report back once I check, but from the looks of it, seems like that's what is needed.

**Comment by nickg** [comment]
Presumably this worked ok.

**Comment by filmil** [comment]
Yes, sorry I forgot to report back - was distracted by trying to compile nvc to be a fully static binary.

The fix worked as expected.

I was, however, unable to make `nvc` into a working fully static binary. :)

**Comment by nickg** [comment]
> 
> I was, however, unable to make `nvc` into a working fully static binary. :)

That probably won't work because the compiled VHDL code needs to dynamically link to symbols from the main binary.

**Comment by filmil** [comment]
> That probably won't work because the compiled VHDL code needs to dynamically link to symbols from the main binary.

Indeed. That is only one of several issues preventing it from happening. This makes nvc more difficult to distribute, than to compile locally.

In the end I fell back to local compilation.
